pkg/hintrunner/zero/zerohint_usort_test.go

```diff
@@ -13,15 +15,14 @@ func TestZeroHintUsort(t *testing.T) {
 		"UsortEnterScope": {
 			{
 				ctxInit: func(ctx *hinter.HintRunnerContext) {
-					err := ctx.ScopeManager.AssignVariable("__usort_max_size", feltUint64(1))
-					if err != nil {
-						t.Fatal(err)
-					}
+					ctx.ScopeManager.EnterScope(map[string]any{
+						"__usort_max_size": uint64(1),
+					})
 				},
 				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
 					return newUsortEnterScopeHint()
 				},
-				check: varValueInScopeEquals("__usort_max_size", feltUint64(1)),
+				check: varValueInScopeEquals("__usort_max_size", uint64(1)),
```

Comment:
is the cast required?

Reply:
nope, removed

Reply:
I guess, since we specify in the scope testing logic that it needs to detect felt, big.int or uint64